src/App.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  AppBar,
+  Alert,
+  Box,
+  Button,
+  Container,
+  FormGroup,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  Toolbar,
+  Typography,
+} from "@mui/material";
+import SendIcon from "@mui/icons-material/Send";
+import React from "react";
+import CodeMirror from "@uiw/react-codemirror";
+import { oneDark } from "@codemirror/theme-one-dark";
+import init, { program } from "miden-wasm";
+import "./App.css";
+
+async function getExample(example: string[], inputs: Boolean) {
+  if (inputs) {
+    const response = fetch(`https://raw.githubusercontent.com/0xPolygonMiden/examples/main/examples/${example}.inputs`)
+    return (await response).text();
+  }
+  else {
+    const response = fetch(`https://raw.githubusercontent.com/0xPolygonMiden/examples/main/examples/${example}.masm`)
+    return (await response).text();
+  }
+}
+
+function App() {
+
+  const [inputs, setInputs] = React.useState(
+    `{
+      "stack_init": ["0"],
+      "advice_tape": ["0"]
+}`
+  );
+
+  var exampleCode =`begin
+  push.1
+  push.2
+  add
+end`
+
+  const [code, setCode] = React.useState(exampleCode);
+
+  const [output, setOutput] = React.useState({
+    init: false,
+    success: true,
+    text: "",
+  });
+
+  const [numOfOutputs, setNumOfOutputs] = React.useState(1);
+
+  // this should be dyncamically read from ./examples/ - buy my TS knowledge is limited
+  const examples = [
+    'collatz',
+    'comparison',
+    'conditional',
+    'fibonacci',
+    'game-of-life-4x4',
+    'nPrime',
```

Comment:
The nPrime example seems to be missing from the examples directory. We should either add it in the examples or remove the entry from here.